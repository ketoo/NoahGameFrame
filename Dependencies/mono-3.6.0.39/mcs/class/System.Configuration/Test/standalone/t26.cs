using System;
using System.Collections.Specialized;
using System.Configuration;
using System.Xml;

class T1
{
	static void Main(string[] args)
	{
		try {
			System.Configuration.Configuration config = ConfigurationManager.OpenExeConfiguration (ConfigurationUserLevel.None);

			ConfigurationSection connStrings = config.ConnectionStrings;

			Console.WriteLine ("connStrings[LocalSqlServer] = {0}", ((ConnectionStringsSection)connStrings).ConnectionStrings["LocalSqlServer"]);

			connStrings.SectionInformation.UnprotectSection ();
			connStrings.SectionInformation.ForceSave = true;
			config.SaveAs ("t27.exe.config", ConfigurationSaveMode.Full);

			if (connStrings.SectionInformation.IsProtected == true)
				Console.WriteLine ("Section {0} is now protected by {1}",
						   connStrings.SectionInformation.Name,
						   connStrings.SectionInformation.ProtectionProvider.Name);
			else
				Console.WriteLine ("Section {0} is not protected", connStrings.SectionInformation.Name);
		}
		catch (Exception e) {
			Console.WriteLine ("{0} raised", e.GetType());
		}
	}
}
