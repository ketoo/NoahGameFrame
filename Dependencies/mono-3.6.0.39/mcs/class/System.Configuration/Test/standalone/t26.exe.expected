connStrings[LocalSqlServer] = data source=127.0.0.1; Integrated Security=SSPI
Section connectionStrings is not protected
